tox.ini

```diff
@@ -37,10 +37,10 @@ setenv =
     dev: MAILCHIMP_USER_ACTIONS_SUBACCOUNT = {env:MAILCHIMP_USER_ACTIONS_SUBACCOUNT:devdata}
     tests: ELASTICSEARCH_INDEX = {env:ELASTICSEARCH_INDEX:hypothesis-tests}
     functests: ELASTICSEARCH_INDEX = {env:ELASTICSEARCH_INDEX:hypothesis-functests}
+    functests: PYTEST_ADDOPTS = {env:PYTEST_ADDOPTS:--reverse}
     {tests,functests}: AUTHORITY = {env:AUTHORITY:example.com}
 passenv =
     HOME
-    PYTEST_ADDOPTS
```

Comment:
Previously any `PYTEST_ADDOPTS` defined in the user's shell was being passed through (to both the unittests and functests) but with no default value.